app/routes/events/list.js

```diff
@@ -88,6 +88,11 @@ export default class extends Route.extend(EmberTableRouteMixin) {
       'page[size]'   : params.per_page || 10,
       'page[number]' : params.page || 4
     };
+    // add condition for default sorting the table by date
+    if(!params.sort_by){
+    params.sort_by = 'starts-at';
+    params.sort_dir = 'DSC';
```

Comment:
Codacy found an issue: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6396347)